components/crud-web-apps/jupyter/frontend/cypress/e2e/form-page.cy.ts

```diff
@@ -57,4 +57,61 @@ describe('New notebook form', () => {
           });
       });
   });
+
+  it('should update panel header name according to the name input field', () => {
+    cy.visit('/new');
+
+    cy.get('[data-cy="workspace volume"]').click();
+
+    cy.get('[data-cy="workspace volume"]')
+      .find('[data-cy="volume name input"]')
```

Comment:
I think that the `cy.get().find()` chain is redundant here and you could convert this to `cy.get('[data-cy="volume name input"]')`. 

The same goes for all `cy.get().find()` in the tests introduced. There are cases where we might need to chain `cy.get()` with `.find()` but as far as I 'm concerned, this is not the case here. WDYT @elenzio9 ?

Reply:
@orfeas-k Thanks for the comments! Just pushed a fixup commit. Please take a look!